packages/node_modules/@webex/plugin-meetings/src/members/util.js

```diff
@@ -156,4 +160,59 @@ MembersUtil.getTransferHostToMemberRequestParams = (options) => {
   };
 };
 
+MembersUtil.genderateSendDTMFOptions = (url, tones, memberId, locusUrl) => ({
+  url,
+  tones,
+  memberId,
+  locusUrl
+});
+
+MembersUtil.generateSendDTMFRequestParams = ({
+  url, tones, memberId, locusUrl
+}) => {
+  const body = {
+    device: {
+      url
+    },
+    memberId,
+    dtmf: {
+      correlationId: uuid.v4(),
+      tones,
+      volume: 5,
```

Comment:
Where did these hardcoded values come from?
```
volume: 5,
durationMillis: 10,

Reply:
https://confluence-eng-gpk2.cisco.com/conf/pages/viewpage.action?spaceKey=LOCUS&title=WEBEX-189819+%3A+Host+is+using+VoIP+audio+for+invite+by+phone